src/document/mupdf_sys.rs

```diff
@@ -33,7 +33,8 @@ pub enum FzSeparations {}
 pub enum FzImage {}
 
 #[link(name="mupdf")]
-#[link(name="mupdfwrapper")]
+#[link(name="mupdf_wrapper", kind="static")]
```

Comment:
This isn't necessary.

Reply:
No, but it is self documenting and sort of indicate to the reader that "hey, something is going on". As some sort of helper, `mupdf_wrapper` is not a _"real" library_, in opposition to `mupdf`, and it's probably a good thing to indicate that in the code; even more so when both are side by side.

Reply:
Agreed.